From aa24c00d9ed1ea9730c0e8c5dccc1bbb5b61e312 Mon Sep 17 00:00:00 2001
From: Lenka Doudova <lryznaro@redhat.com>
Date: Mon, 26 Jan 2026 16:21:23 +0100
Subject: [PATCH] Issue 7014 - memberOf - ignored deferred updates with LMDB

Description:
Fix typo in pytest marker reason.

Relates: #7014

Author: Lenka Doudova
Reviewed by: ???
---
 .../suites/memberof_plugin/memberof_deferred_lmdb_test.py      | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dirsrvtests/tests/suites/memberof_plugin/memberof_deferred_lmdb_test.py b/dirsrvtests/tests/suites/memberof_plugin/memberof_deferred_lmdb_test.py
index 0d9f793c1..12fcfa3ec 100644
--- a/dirsrvtests/tests/suites/memberof_plugin/memberof_deferred_lmdb_test.py
+++ b/dirsrvtests/tests/suites/memberof_plugin/memberof_deferred_lmdb_test.py
@@ -1,4 +1,5 @@
 # --- BEGIN COPYRIGHT BLOCK ---
+
 # Copyright (C) 2025 Red Hat, Inc.
 # All rights reserved.
 #
@@ -28,7 +29,7 @@ else:
     logging.getLogger(__name__).setLevel(logging.INFO)
 
 
-@pytest.mark.skipif(get_default_db_lib() != "mdb", reason="Not supported over mdb")
+@pytest.mark.skipif(get_default_db_lib() != "mdb", reason="Not supported over bdb")
 def test_memberof_deferred_update_lmdb_rejection(topo):
     """Test that memberOf plugin rejects deferred update configuration with LMDB backend
 
-- 
2.52.0

